dashboard/src/components/AppView/AppView.tsx

```diff
@@ -152,7 +152,7 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
       return <div>Loading</div>;
     }
     return (
-      <section className="AppView padding-b-big">
+      <section id="appview" className="AppView padding-b-big">
```

Comment:
do we need this?

Reply:
guessing you added this for the Modal.setAppElement thing, but ultimately decided to set ariaHideApp to false?